change/@fluentui-react-overflow-7c62c2b9-c455-4726-8229-490cfdbb5fc1.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "chore: update api.md",
```

Comment:
affected apis that contain the emit change because of `exactOptionalPropertyTypes` emit change need to bump as patch.

Additionally we need to explicitly document affected packages in the PR description and improve the change comment so we have reference point for partners in order to communicate this.

in future once we land a custom `Changelog` renderer within our docs it will be highlighted there.